### PR TITLE
Bug fix on GalleryContainer.js.

### PR DESCRIPTION
**Pull request by ashtilawat**
Overview of Changes:

- Fixed bug on GalleryContainer.js. Added props as a parameter on line 11. 

Why was this done? 

- App was breaking after a fresh pull from main. This was necessary to get it working again.

### CODE REVIEW

**Review by Alisa1989 — APPROVED** [review]

**Review by rhiannonstanford — APPROVED** [review]
Thank you.  Change looks good.

File: src/components/pages/Gallery/GalleryContainer.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { setWeeklySubmissions } from '../../../state/actions/galleryActions';
 import WeeklySubmissions from './WeeklySubmissions';
 import { useHistory, useParams } from 'react-router-dom';
 
-const GalleryContainer = () => {
+const GalleryContainer = props => {
```
Comment (just-kato) [inline src/components/pages/Gallery/GalleryContainer.js:11]: Niceeee props to save the day!

**Review by milklot — COMMENTED** [review]
Good job on  time saving for everyone

**Comment by tonysorensen** [comment]
Props on the props.

**Pull request merged**